angularSTEP/package.json

```diff
@@ -33,13 +33,12 @@
     "@types/googlemaps": "^3.39.9",
     "@types/jasmine": "^3.5.11",
     "@types/karma": "^5.0.0",
-    "@types/mocha": "^8.0.0",
     "algoliasearch": "^3.35.1",
     "angular-instantsearch": "^3.0.0-beta.5",
     "firebase": "^7.15.5",
     "instantsearch.css": "^7.4.2",
     "instantsearch.js": "^3.7.0",
-    "mocha": "^8.0.1",
```

Comment:
Are you sure "mocha" isn't needed/used anywhere?

Reply:
Yes, it is not needed